src/scenarios/genericandroidstartup/pre.py

```diff
@@ -0,0 +1,49 @@
+'''
+pre-command
+'''
+import sys
+import os
+from zipfile import ZipFile
+from performance.logger import setup_loggers, getLogger
+from shutil import copyfile
+from shared.precommands import PreCommands
+from shared.const import PUBDIR
+from argparse import ArgumentParser
+
+setup_loggers(True)
+
+parser = ArgumentParser()
+parser.add_argument('--unzip', help='Unzip APK and report extracted tree', action='store_true', default=False)
```

Comment:
Do we need the unzip logic if this scenario is only about startup measurements ?

Reply:
Updated as we shouldn't need it.